examples/complete/main.tf

```diff
@@ -20,6 +20,17 @@ module "vpc" {
 
   assign_generated_ipv6_cidr_block = true
 
+  adopt_default_network_acl    = true
+  adopt_default_route_table    = true
+  adopt_default_security_group = true
+
+  context = module.this.context
+}
+
+module "vpc_defaults" {
```

Comment:
Instead of creating another instance of it in the same root module, we may want to expose the `adopt_` inputs in example/complete, set them to `false` in the current test fixtures tfvars, and create a new set of test fixtures tfvars to use the `_adopt.* = true` and a corresponding terratest.